UI.md

```diff
@@ -0,0 +1,71 @@
+# AWS Lambda Power Tuning UI
+
+- [Basic Demo](https://digio.github.io/aws-lambda-power-tuning-ui/#gAEAAgAEAAYACA==;Go9qRqDLVUbymEpGgC0jRn/iMkY=;41bGOHAK8Th8bWQ5mvyJObyvyTk=;)
+- [Comparison Demo](https://digio.github.io/aws-lambda-power-tuning-ui/#gAEAAgAEAAYACA==;Go9qRqDLVUbymEpGgC0jRn/iMkY=;41bGOHAK8Th8bWQ5mvyJObyvyTk=;gAEAAgAEAAYACA==;0+hrRh7TR0YDoxBG/mQCRjclCkY=;sX2hOBZhtji8AgQ5ZIcyOV8vfDk=;JS%20x86;JS%20ARM64)
+
+The website is deployed to the `gh-pages` branch of this repo, which is hosted at [https://digio.github.io/aws-lambda-power-tuning-ui/](https://digio.github.io/aws-lambda-power-tuning-ui/)
```

Comment:
The domain https://lambda-power-tuning.show/ is an Amplify Hosting app. While I don't see any major problems with deploying a sample version on gh-pages, I would not recommend that for "production".